conrad-blucher-institute/Flare
# 15 write logic for laguna madre page

This pull request focuses on the design of the South Bird Island chart to closely mimic the layout of the [TPW page](https://cbigrid.tamucc.edu/tpw/) as requested.

## Testing Instructions

1. Update the nginx config
    - I will send an encrypted email showing what needs to be replaced
3. Open two WSL terminals side by side.
     - In one terminal, start the Docker containers.
     - In the other terminal, run the development server.
  
4. Once in the development server click the Turtle button to display the chart.
5. Verify that the chart graphs as expected.
## Notes:
- Gaps in the Chart: Any gaps in the chart are intentional and caused by the 10-hour interval in the test data, which was set to maintain a clean and uncluttered visualization during testing.

- Generating Test Data: You can generate test CSV data by running this command: python3 backend/tests/test_csv_generator.py
